### PR TITLE
FPLA-120: Add-PBA-Number

### PR DESCRIPTION
**Pull request by brdgeth**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPLA-120

### Change description ###
Add PBA number when entering an Applicant.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] No

### CODE REVIEW

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                .content(MAPPER.writeValueAsString(request)))
+            .andExpect(status().isOk())
+            .andReturn();
+
+        return MAPPER.readValue(response.getResponse()
+            .getContentAsByteArray(), AboutToStartOrSubmitCallbackResponse.class);
+    }
+}
+
+
```
Comment (Jamesnelson117) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java]: Remove white space, we keep one line space at the end of files only. I'm surprised the linter did not flag.

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        // then
+        assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
+        String actualPbaNumber = extractPbaNumberFromApplicant(callbackResponse);
+        assertThat(actualPbaNumber).isEqualTo("PBA1234567");
+    }
+
+    @Test
+    void shouldReturnNoErrorsWhenPbaNumberIsNotEntered() throws Exception {
+        // given
```
Comment (Jamesnelson117) [inline service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java]: Remove comments

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                caseDetails.getData().put("applicant", applicantData);
+            }
+        }
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(caseDetails.getData())
+            .errors(validationErrors)
+            .build();
+    }
+
```
Comment (Jamesnelson117) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java]: Remove white space

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java [hunk 4]
```diff
@@ -44,6 +44,9 @@ void shouldGenerateSubmittedFormDocumentWhenCaseIsFullyPopulated() throws IOExce
         String content = textContentOf(createServiceInstance(clock).generateSubmittedFormPDF(populatedCaseDetails(),
             Pair.of("userFullName", "Emma Taylor"))
         );
+
+        System.out.println(content);
```
Comment (Jamesnelson117) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java]: Remove logging please. But this was my fault as you where helping me with debugging PDF

**Comment by brdgeth** [comment]
Thank you for looking at this.  I have removed the comments and whitespace from the code.

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java (hunk 3 above)
Comment (brdgeth) [reply service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java]: Done

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java (hunk 1 above)
Comment (brdgeth) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java]: Done

File: service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java (hunk 2 above)
Comment (brdgeth) [reply service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantControllerTest.java]: Done

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java (hunk 4 above)
Comment (brdgeth) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentGeneratorServiceTest.java]: Done

**Comment by SamanthaSJ** [comment]
Verified in PR-300 Pass.

**Pull request merged**